### PR TITLE
Feature/openzipkin

### PR DESCRIPTION
**Pull request by PhilYue**
Topic: Zipkin & Prometheus in Dubbo-go Example
Description:
- full call chain tracing implementation
- .run config
- environment is based on Docker
- two versions of README

### CODE REVIEW

**Review by zouyx — APPROVED** [review]

File: openzipkin/README.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+# Zipkin in Dubbo-go Example
+
+## Backend
+
+Zipkin is a distributed tracing system. It helps gather timing data needed to troubleshoot latency problems in service architectures. Features include both the collection and lookup of this data.
```
Comment (AlexStocks) [inline openzipkin/README.md:5]: 改为后面内容即可：Zipkin is a distributed tracing system. It helps dubbo-go gathering timing data which is used to troubleshoot latency problems in service architectures.

File: openzipkin/README.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+services:
+  ...
+filter: "tracing"
+
+```
+
+## Filter
+
+Dubbo-go support `opentrace filter` 实现，基于简单配置即可
```
Comment (zhaoyunxing92) [inline openzipkin/README.md:77]: 换成英语

File: openzipkin/README.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+```
+
+- go-client ：The Service Consumer
+- go-server-a ：The Service Provider A
+- go-server-b ：The Service Provider B
+
+Client calls ProviderA , and then calls ProviderB,
+ProviderA also makes a request to ProviderB！
+
+依赖关系：
```
Comment (AlexStocks) [inline openzipkin/README.md:27]: dependency hierarchy:

File: openzipkin/README.md (hunk 2 above)
Comment (AlexStocks) [inline openzipkin/README.md:77]: Dubbo-go supports `opentrace filter`.

**Review by zhaoyunxing92 — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

**Pull request merged**